UPGRADE-0.x.md

```diff
@@ -1,2 +1,10 @@
 UPGRADE 0.x
 ===========
+
+UPGRADE FROM 0.4 to 0.x
```

Comment:
We have no stable release yet, so we can remove the SonataEasyExtends bundle with a new *unstable* minor release.